src/lib/components/billing/alerts/selectProjectCloud.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+    import { type Models } from '@appwrite.io/console';
+    import { Alert, Button, Table } from '@appwrite.io/pink-svelte';
+    import { Modal } from '$lib/components';
+    import { onMount } from 'svelte';
+    import { sdk } from '$lib/stores/sdk';
+    import { addNotification } from '$lib/stores/notifications';
+    import { invalidate } from '$app/navigation';
+    import { Dependencies } from '$lib/constants';
+    import { billingProjectsLimitDate } from '$lib/stores/billing';
+    import { page } from '$app/state';
+    import { toLocaleDateTime } from '$lib/helpers/date';
+
+    export let showSelectProject: boolean;
+    export let selectedProjects: Array<string> = [];
```

Comment:
lets move to runes here as well, for binding a var, use `$bindable` and `$props` instead of `export let`.